<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>178</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>178</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>178</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Параметры окна демонстарции</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../IMGs/monitor.ico</normaloff>../IMGs/monitor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>5</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>-20</y>
      <width>250</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Демонстарция&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMGs/setting.png</normaloff>../IMGs/setting.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>132</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
